gitops/github/workflow.yml

```diff
@@ -0,0 +1,55 @@
+on:
+  push:
+    branches:
+      - main
+      - master
+  
+  pull_request:
+    branches:
+      - main
+      - master
+
+jobs:
+  FabrikateToManifest:
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download Bedrock orchestration script
+        run: |
+          curl $BEDROCK_BUILD_SCRIPT > build.sh
+          chmod +x ./build.sh
+        shell: bash
+        env:
+          BEDROCK_BUILD_SCRIPT: https://raw.githubusercontent.com/Microsoft/bedrock/master/gitops/azure-devops/build.sh
+
+  
+      - uses: azure/setup-helm@v1
+        with:
+          version: '2.17.0' # default is latest stable
```

Comment:
I thought we were using Helm v3?

Reply:
helm v3 requires even more changes, such as updating the build.sh file to remove the 'init helm' sections.
helm v2.17 is the best tradeoff, as it uses the new charts uris while at the same time doesn't require additional changes which I didn't want to introduce to this pr